src/compiler/output-targets/dist-custom-elements/test/dist-custom-elements.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { addCustomElementInputs } from '../index';
+
+describe('dist-custom-elements', () => {
+  it('should add custom element inputs', () => {
+    const buildCtx: any = {
+      components: [
+        {
+          isPlain: true,
+          sourceFilePath: './foo/bar.tsx',
+          tagName: 'my-tag',
+        },
+      ],
+    };
+    const bundleOpts: any = {
```

Comment:
Note: This is included in the patch file in my first comment on this PR

I think this could be made a `BundleOptions` object by adding an `id` and `platform` property here:
```ts
    const bundleOpts: BundleOptions = {
      id: 'customElements',
      platform: 'client',
      inputs: {},
      loader: {},
    };
```